### PR TITLE
Remove unused variable

### PR DESCRIPTION
**Pull request by prachigarg19**
Remove unused variables across webapp

### CODE REVIEW

File: src/features/common/Layout/Footer/index.tsx [hunk 1]
```diff
@@ -90,7 +89,6 @@ export default function Footer(): ReactElement | null {
       }
       if (localStorage.getItem('language')) {
         const langCode = localStorage.getItem('language');
-        if (langCode) setLanguage(langCode);
       }
```
Comment (mohitb35) [inline src/features/common/Layout/Footer/index.tsx:92]: This code is unused as well now.

File: src/features/common/Layout/Footer/AccountFooter.tsx
Comment (mohitb35) [inline src/features/common/Layout/Footer/AccountFooter.tsx:1]: Is this component used any more?

File: src/features/projects/components/projects/Filters.tsx [hunk 2]
```diff
@@ -11,10 +11,10 @@ export default function Filters(): ReactElement {
   const { projects, setFilteredProjects, filtersOpen, setFilterOpen } =
     useProjectProps();
 
-  const [purpose, setPurpose] = React.useState({
+  const purpose = {
     restoration: true,
     conservation: true,
-  });
+  };
```
Comment (mohitb35) [inline src/features/projects/components/projects/Filters.tsx:17]: `purpose` does not seem required any more - @prachigarg19 please check and remove if not needed.

File: src/features/user/Profile/components/MyTrees/MyTreesMap.tsx
Comment (mohitb35) [inline src/features/user/Profile/components/MyTrees/MyTreesMap.tsx:1]: Please remove unused props where MyTreesMap is imported as well .. e.g. src/features/user/Profile/components/MyTrees/MyTrees.tsx

File: src/features/user/RegisterTrees/RegisterTreesWidget.tsx [hunk 3]
```diff
@@ -422,7 +422,7 @@ type FormData = {
 
 export default function RegisterTreesWidget() {
   const { user, token } = useUserProps();
-  const { t, ready } = useTranslation(['me', 'common']);
+  const { ready } = useTranslation(['me', 'common']);
```
Comment (mohitb35) [inline src/features/user/RegisterTrees/RegisterTreesWidget.tsx:425]: It does not appear like we need `ready` either, if we are not using translations in any way. Please remove if not needed.

File: src/features/user/TreeMapper/components/Map.tsx
Comment (mohitb35) [inline src/features/user/TreeMapper/components/Map.tsx:1]: `getDateDiff` does not seem to be needed any more, please remove if unused.

**Review by mohitb35 — CHANGES_REQUESTED** [review]
@prachigarg19 Please go through the minor comments below

File: src/features/common/Layout/Footer/AccountFooter.tsx
Comment (prachigarg19) [reply src/features/common/Layout/Footer/AccountFooter.tsx:1]: I don't think so. I'll delete this file.

File: src/features/common/Layout/Footer/index.tsx (hunk 1 above)
Comment (mohitb35) [reply src/features/common/Layout/Footer/index.tsx]: This was missed, resolved here > [9dd8228](https://github.com/Plant-for-the-Planet-org/planet-webapp/pull/1890/commits/9dd822814f6c3cfa51524d820a815478f2d632c4)

**Review by mohitb35 — APPROVED** [review]

**Review by mohitb35 — APPROVED** [review]
LGTM

**Pull request merged**